### ISSUE TITLE
pure maven project call gradle build.

### ISSUE THREAD

**Issue by FinallyFinancialFreedom** [artur-shaik/vim-javacomplete2#320]
## Actual behavior (Required!)
in a pure maven project, when i open App.java, this plugin call gradle build at first, and sure it failed. then it told me gradle is build successfully. and then it told me maven is build successfully.

## Expected behavior (Required!)
do not call gradle build

## The steps to reproduce actual behavior (Required!)
 1. unzip my uploaded pure maven java project.
[case-maven.zip](https://github.com/artur-shaik/vim-javacomplete2/files/891903/case-maven.zip)

 2. vi open App.java
 3. the first output screenshot:
<img width="1194" alt="screen shot 2017-04-04 at 10 02 34 am" src="https://cloud.githubusercontent.com/assets/2133400/24638965/1c78cbce-191f-11e7-91cf-a4477fdc1995.png">
when i press enter, the second output screenshot:
<img width="1172" alt="screen shot 2017-04-04 at 10 02 51 am" src="https://cloud.githubusercontent.com/assets/2133400/24638969/2aecfc52-191f-11e7-8095-f1fae9055b0b.png">

## Environment (Required!)
 * OS:MacOS
 * Vim version:
 * Neovim version:0.1.7

## Q&A
* Yes, I tried minimal .vimrc configuraion.
* Yes, I have enabled logs with `JCdebugEnableLogs` and can put here content of `JCdebugGetLogContent` command, if you need.
* Even, if you wish, I can set `g:JavaComplete_JavaviDebug` to `1`, than set `g:JavaComplete_JavaviLogfileDirectory`, and put here server logs, too.

**Comment by wsdjeg** [comment]
I will test it, I think it is due to gradle project check.

**Comment by wsdjeg** [comment]
after `mvn compile test`, you will see
```
wsdjeg@archlinux:~/Downloads/case-maven$ la -l
total 16
drwxr-xr-x 4 wsdjeg wsdjeg 4096 Apr  4 10:02 .gradle
-rw-r--r-- 1 wsdjeg wsdjeg  750 Apr  4 10:01 pom.xml
drwxr-xr-x 4 wsdjeg wsdjeg 4096 Apr  4 10:01 src
drwxr-xr-x 6 wsdjeg wsdjeg 4096 Apr  4 10:43 target
```

I think there should not be `.gradle` dir, I think it is issue with pom.xml

**Comment by FinallyFinancialFreedom** [comment]
yes, i can see .gradle dir in my computer, but it's created by javacomplete2 after i open App.java with vi.
if i don't open it with vi, `mvn compile test` will not create .gradle dir.

**Comment by artur-shaik** [comment]
I cannot reproduce it with your project. No `.gradle` directory.
@wsdjeg, there is `.gradle` directory in archive, remove it before test.

**Comment by artur-shaik** [comment]
@FinallyFinancialFreedom, and by the way, look like those errors dues to some permission problem in your system.

**Comment by FinallyFinancialFreedom** [comment]
@artur-shaik thank you for your test, I figured out what really happened.
I use gradle 3.4.1 and there is a problem, I have opened a issue #319 about it.
I thought the problem of  #319  is the gradle config so I add `let g:JavaComplete_GradleExecutable = 'gradle' ` to use my preinstalled gradle, that's the reason of this issue. now i clear it and the problem is gone. Maybe i misused this config option? It always triggers the gradle build even in a pure maven project.
